api/accesslog.proto

```diff
@@ -169,3 +169,30 @@ message EnvoyAccessLog {
   google.protobuf.UInt32Value tls_cipher_suite = 30;
 }
 
+// Stream message for the StreamAccessLogs API. Envoy will open a stream to the server and stream
+// access logs without ever expecting a response.
+message EnvoyAccessLogMessage {
+  // The node sending the access log messages over the stream. Note that the server should only
+  // expect node to be populated on the first message in the stream. It will not be populated in
+  // subsequent messages. This is a performance optimization.
+  Node node = 1;
+  // The individual access log message.
+  EnvoyAccessLog log = 2;
+}
+
+// Empty response for the StreamAccessLogs API. Will never be sent. See below.
+message EnvoyAccessLogResponse {}
+
+// Service for streaming access logs from Envoy to an access log server.
+service EnvoyAccessLogService {
```

Comment:
pardon the newb question: is there a reason to prepend "Envoy" to all the message/service names?  The package name already includes "envoy".

Reply:
No reason. I was just copying what was already in the file, but I agree it's not useful and since no one is using the existing proto yet I will change all the names.

Reply:
+1